frontend/src/pages/Administration.js

```diff
@@ -32,7 +16,7 @@ export default function Administration() {
         <TableContainer>
           <Typography variant="h5">Notification User</Typography>
           <StyledDataGrid
```

Comment:
be careful with the usage of mui datagrid. There's a free version and another one that has to be licensed